src/k8s-engine/lima.ts

```diff
@@ -330,6 +331,48 @@ export default class LimaBackend extends events.EventEmitter implements K8s.Kube
     return Promise.resolve(null);
   }
 
+  /**
+   * Check if the base (alpine) disk image is out of date; if yes, update it
+   * without removing existing data.  This is only ever called from updateConfig
+   * to ensure that the passed-in lima configuration is the one before we
+   * overwrote it.
+   */
+  protected async updateBaseDisk(currentConfig: LimaConfiguration) {
+    // Lima does not have natively have any support for this; we'll need to
+    // reach into the configuration and:
+    // 1) Figure out what the old base disk version is.
+    // 2) Confirm that it's out of date.
+    // 3) Change out the base disk as necessary.
+    // Unfortunately, we don't have a version string anywhere _in_ the image, so
+    // we will have to rely on the path in lima.yml instead.
+
+    const images = currentConfig.images.map(i => path.basename(i.location));
+    // We had a typo in the name of the image; it was "alpline" instead of "alpine".
+    const versionMatch = images.map(i => /^alpl?ine-lima-v([0-9.]+)-rd-/.exec(i)).find(defined);
```

Comment:
I think this was still the `std` edition in RD 0.4.1 (not sure if non-capturing groups are supported here):

```suggestion
    const versionMatch = images.map(i => /^alpl?ine-lima-v([0-9.]+)-(?:std|rd)-/.exec(i)).find(defined);
```

Reply:
I just tried an update from 0.4.1 to this PR. My logs have the message...

```
Could not find base image version from alpline-lima-v0.1.0-std-3.13.5.iso; skipping update of base images.
```

Because the images wasn't updated things like nerdctl are not on it.

This change fixed the issue. I would commit it but I don't think that works for the DCO check.

Reply:
Fixed in 17ee47b463d7cd061e7129b62596d17c9ce84f84 (just dropped the whole chunk).

I believe DCO would still be okay, if you just appended a commit (with DCO) on top of the branch.